### DISCUSSION TITLE
K8s 1.23 - Formalize docs release strategy with docs localization owners

### DISCUSSION THREAD

**Discussion by jlbutler** [kubernetes/sig-release#1686] [General]
Hello 👋, I'm Jesse Butler, the SIG Release Docs lead for the 1.23 release. I'm opening this discussion to coordinate the docs release with localization owners. The 1.23 release is scheduled tentatively for Tuesday, December 7, please see the [release timeline](https://github.com/kubernetes/sig-release/tree/master/releases/release-1.23#tldr).

I look forward to working with everyone! Let me know if you have any questions! Thank you! 🙏

cc: @kubernetes/sig-docs-fr-owners @kubernetes/sig-docs-de-owners @kubernetes/sig-docs-hi-owners @kubernetes/sig-docs-id-owners @kubernetes/sig-docs-ja-owners @kubernetes/sig-docs-ru-owners @kubernetes/sig-docs-pl-owners @kubernetes/sig-docs-ko-owners @kubernetes/sig-docs-pt-owners @kubernetes/sig-docs-es-owners @kubernetes/sig-docs-zh-owners @kubernetes/sig-docs-vi-owners @kubernetes/sig-docs-uk-owners @kubernetes/sig-docs-it-owners

**Comment by raelga** [comment]
Count me in for `sig-docs-es`.

**Comment by micheleberardi** [comment]
Count me in for`sig-docs-it`

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by seokho-son** [comment]
I am here for `sig-docs-ko`. Thank you !

**Comment by inductor** [comment]
ping me for `sig-docs-ja`

**Comment by rikatz** [comment]
Here as well for docs-pt :)

**Comment by fabriziopandini** [comment]
I'm in for IT

**Comment by MaxymVlasov** [comment]
Count me for `sig-docs-uk`

**Comment by jlbutler** [comment]
Thanks all! 🚀

**Comment by mittalyashu** [comment]
Add me for `sig-docs-hi`

**Comment by jlbutler** [comment]
Hi localization team leads! I wanted to let you know that we are on track for the 1.23 release tomorrow December 7, 2021. Later today (later afternoon PDT) we will be freezing k/website in preparation for the release tomorrow. I don't believe there are any actions required from you, but please let me know if there's anything I can help with or if anyone has any issues. Thanks!

**Comment by huynguyennovem** [comment]
Count me for `sig-docs-vi`. Thanks!
